torch/_inductor/lowering.py

```diff
@@ -928,6 +928,15 @@ def as_strided_copy(x, size, stride, storage_offset=None):
 
 @register_lowering(aten.cat)
 def cat(inputs, dim=0):
+    if all(input.get_dtype() is torch.uint8 for input in inputs):
```

Comment:
would it make more sense to register a separate, quantized cat op? Is this different than `quantized::cat` ?

Reply:
yean, I think `aten::cat` already cover this functionality and make the least changes. Since `quantized::cat` assume inputs as `quantized tensor`, we need to make extra change to the implementation of `quantized::cat` to make it support of `plain tensor` with `uint8` data type.